proprietary/design-tokens/src/imported/product/NLdoc-mapping.json

```diff
@@ -86,6 +86,11 @@
           "parent": "viewport (beta)/xs",
           "type": "lineHeights",
           "value": "{lux.font-size.heading.level-4}*1.5"
+        },
+        "level-5": {
+          "parent": "viewport (beta)/xs",
+          "type": "lineHeights",
+          "value": "{lux.font-size.heading.level-5}*1.5"
```

Comment:
Het is beter om spaties om de `*` te zetten, aangezien StyleDictonary er anders niet mee om kan gaan.

Reply:
Dan moeten we dat doorvoeren op alle regels ;) 

Ik heb het blokje namelijk ge-copy pasta'd